doc/wiki/running-ab-experiments.md

```diff
@@ -3,39 +3,59 @@
 ## Experiment setup:
 - experiments can be set up by adding a data migration to `db/data`
 - an example of how to set up an experiment can be found in `lib/seed/experiment_seeder.rb`
+- the `experiment` feature flag must be on before the patient selection process and while experiment notifications are still sending. Turning it off will prevent patient selection and experiment notifications from being scheduled to send each day.
```

Comment:
Alot of this doc looks like it duplicates the release plan we moved into the repo -- maybe just link to that with something like "consult the docs for our initial launch to see how to run an experiment" ?